src/storage/Manager.cc

```diff
@@ -48,24 +50,36 @@ void Manager::InitPostScript() {
     StartExpirationTimer();
 }
 
-zeek::expected<BackendPtr, std::string> Manager::Instantiate(const Tag& type) {
-    Component* c = Lookup(type);
-    if ( ! c ) {
+zeek::expected<BackendPtr, std::string> Manager::Instantiate(const Tag& btype, const Tag& stype) {
```

Comment:
It could make sense to split out `InstantitateSerializer(tag)` as an API so other places could get serializers. This might be a bit far out there, but that way a bif like the following becomes easy:
```
function serialize_val(tag: Storage::Serializer, v: any): string
```
Probably mostly useful for testing, but hey :-)